cmd/mountlib/mount.go

```diff
@@ -51,6 +65,111 @@ func init() {
 	}
 }
 
+// AddRc adds mount and unmount functionality to rc
+func AddRc(mountFunction MountFn) {
+	// rcMount allows the mount command to be run from rc
+
+	currentMountFn = mountFunction
+
+	rc.Add(rc.Call{
+		Path:         "mount/mount",
+		AuthRequired: true,
+		Fn:           mountRc,
+		Title:        "Create a new mount point",
+		Help: `
+rclone allows Linux, FreeBSD, macOS and Windows to
+mount any of Rclone's cloud storage systems as a file system with
+FUSE.
+
+This takes the following parameters
+
+	- mountPoint: valid path on the local machine (required)
+	- remotePath - a remote path to be mounted (required)
+
+Eg
+
+    rclone rc mount/mount mountPoint=/home/<user>/mountPoint remotePath=mydrive:
+		`,
+	})
+
+	rc.Add(rc.Call{
+		Path:         "mount/unmount",
+		AuthRequired: true,
+		Fn:           unMountRc,
+		Title:        "Unmount all active mounts",
+		Help: `
+rclone allows Linux, FreeBSD, macOS and Windows to
+mount any of Rclone's cloud storage systems as a file system with
+FUSE.
+
+This takes the following parameters
+
+	- mountPoint: valid path on the local machine (required)
+	- remotePath - a remote path to be mounted (required)
+
+Eg
+
+    rclone rc mount/unmount mountPoint=/home/<user>/mountPoint remotePath=mydrive:
+		`,
+	})
+}
+
+// rcMount allows the umount command to be run from rc
+func unMountRc(_ context.Context, _ rc.Params) (out rc.Params, err error) {
+
+	out = make(rc.Params)
+	if currentUnmountFn != nil {
+		err := currentUnmountFn()
```

Comment:
Just return the err - the handling code will deal with setting the status.

so

```go
err := currentUnmountFn()
if err != nil {
    return nil, err
}
```